io.asgardio.android.oidc.sdk.sample/build.gradle

```diff
@@ -18,7 +18,7 @@
 
 apply plugin: 'com.android.application'
 
-version = '0.0.2-SNAPSHOT'
```

Comment:
Can we inherit this from a base properties, rather defining the version again here?

Reply:
Resolved in https://github.com/wso2-extensions/identity-sdks-android/pull/15/commits/b54171ed692633312b0872c9da7ddf9c18bd5355